modules/compliance-tests/src/smithy4s/compliancetests/internals/Assertions.scala

```diff
@@ -50,8 +51,19 @@ private[internals] object assert {
     }
   }
 
-  def eql[A](expected: A, actual: A): ComplianceResult = {
-    if (expected == actual) {
+  def neq[A: Eq](expected: A, actual: A): ComplianceResult = {
```

Comment:
two minor things, feel free to ignore

1. I think `eq/neq` or `eql/neql` would make better pairs rather than `eql/neq`
2. Is it possible to implement using `eql` and `swap` on `Either`?

Reply:
1. agreed
2. 2. i wouldnt be able to provide the specific error message for expected failure